ci/buildspec_gpu.yml

```diff
@@ -7,9 +7,8 @@ phases:
     commands:
       - apt-get update
       - apt-get install sudo -y
-      - python ts_scripts/install_dependencies.py --cuda=cu102 --environment=dev
+      - pip install -r ci/launcher/requirements.txt
```

Comment:
@nskool Is the new script able to take care of installing the correct cuda dependencies?. It was not very clear how this will be handled now